install/syscalls-tracer-pod.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Pod
```

Comment:
Can this pod trace multiple containers at the same time? Or should we have multiple instances of the same pod for each monitored pod?

Reply:
Yes, the pod can trace multiple containers at the same time.